source/run-command/tests/unified/runCommand.yml

```diff
@@ -67,6 +67,38 @@ tests:
                 $readPreference: { $$exists: false }
               commandName: ping
 
+  - description: always gossips the $clusterTime on the sent command
```

Comment:
> Perhaps it would be worth scripting this so we do not have to maintain duplicate yaml

Since there's just two cases for now, I'd probably punt on that until a third or fourth appeared.

I did note the `runCommand.yml` and `runCursorCommand.yml` files were quite large, so alternatively just consider break them down into smaller files (e.g. `runCommand-clusterTime.yml`). From there, it'd be easier to keep them in sync and also compare at a glance.

Reply:
I assume you don't intend to duplicate the test to `runCursorCommand` or split up the test files, so I'll resolve this thread.